src/main/scala/io/github/davidgregory084/ScalacOptions.scala

```diff
@@ -141,6 +141,11 @@ trait ScalacOptions {
   val lint =
     advancedOption("lint", version => version < V2_11_0)
 
+  /** Enable SIP-22 async/await constructs
+    */
+  val async =
```

Comment:
I agree bincompat isn't particularly important at this moment, but it's also not hard to fix FTR.

```suggestion
  def async =
```